notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -13,6 +13,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    fig.savefig("adv-497mil-vs-cp505h-v2048-vary-adv-visits.pgf", backend="pgf")
</pre><p>Again not related to the PR, but interesting to see that win rate <em>does</em> increase with adversary visits -- just only up to around ~600 where it was trained with. Then it does seem flat and I guess based on your results with v=12800 ends up decreasing later?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>